pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -150,12 +150,16 @@ class VmOverviewTabLibvirt extends React.Component {
                 </button>
             );
         }
-        const vcpuLink = (
-            <>
-                <button className="link-button" id={`${vmId(vm.name)}-vcpus-count`} onClick={this.openVcpu}>{vm.vcpus.count}</button>
-                { vm.state === "running" && vcpusChanged && <WarningInactive iconId="vcpus-tooltip" tooltipId="tip-vcpus" /> }
-            </>
-        );
+
+        let vcpuLink = vm.vcpus.count;
+        if (vm.persistent) {
```

Comment:
I generally prefer to disable the button/link instead of hiding it when the operation is not allowed due to a constraint. (VM is transient).
In any case we should decide if we want disabled or hidden buttons and use one approach. Here we already used disabled for the delete button.

Reply:
Hmm, I think some time ago we had a discussion about similar topic (or @garrett talked about it somewhere), and if I remember correctly:
- We should **disable** the button if there is some way, some action which user can do to enable that button.
- If there is nothing an user can do, then we should **hide** the button.

Does this approach fit?

Reply:
Also this piece of code doesn't hide the vcpu count, it merely disables the link which opens up a configuration modal (in other words, it doesn't add the "link" part into the vcpu count)